File: plugins/claude-md-plugin/core/tests/fixtures/java/ExportCandidates.java
package com.example.auth;

public class ExportCandidates {

    public static final int MAX_RETRIES = 3;
    public static final String API_BASE_URL = "https://api.example.com";
    public static final long DEFAULT_TIMEOUT = 30000L;

    private static final String INTERNAL_SECRET = "secret";

    public boolean processItem(String item) {
        return item != null && !item.isEmpty();
    }
}

File: plugins/claude-md-plugin/core/tests/fixtures/kotlin/ExportCandidates.kt
package com.example.auth

const val MAX_RETRIES = 3
val DEFAULT_TIMEOUT: Long = 30000L
val API_BASE_URL = "https://api.example.com"

typealias UserId = String
typealias TokenMap = Map<String, String>

object AppConfig {
    val version = "1.0.0"
}

object DatabaseManager {
    fun connect() {}
}

interface Validator {
    fun validate(): Boolean
}

interface Serializable {
    fun serialize(): ByteArray
}

fun processItem(item: String): Boolean {
    return item.isNotEmpty()
}
